charts/sonarqube-dce/values.yaml

```diff
@@ -549,8 +549,11 @@ extraConfig:
   configmaps: []
 
 # account:
+# The values can be set to define the current and the (new) custom admin passwords at the startup (the username will remain "admin")
 #   adminPassword: admin
 #   currentAdminPassword: admin
+# The above values can be also provided by a secret that contains "password" and "currentPassword" as keys. You can generate such a secret in your cluster using "kubectl create secret generic admin-password-secret-name --from-literal=password=admin --from-literal=currentPassword=admin"
```

Comment:
Let's wrap this line so it is not too long.
The general rule for SQ is 180

Reply:
yes, just wrapped both comments in standard and dce charts